js/unit_tests.ts

```diff
@@ -92,23 +92,27 @@ test(async function tests_readFileSync() {
   assertEqual(pkg.name, "deno");
 });
 
-/*
-test(async function tests_fetch() {
-  const response = await fetch("http://localhost:4545/package.json");
-  const json = await response.json();
-  assertEqual(json.name, "deno");
-});
-
 test(async function tests_writeFileSync() {
   const enc = new TextEncoder();
   const data = enc.encode("Hello");
   // TODO need ability to get tmp dir.
-  const fn = "/tmp/test.txt";
-  writeFileSync("/tmp/test.txt", data, 0o666);
-  const dataRead = readFileSync("/tmp/test.txt");
+
+  // TODO @NewLunarFire: This test currently generates a random directory
+  // name from a random Number converted to String. This should be replaced
+  // with creating an actual tmpdir in the future
+  const randomSuffix = Math.random().toString(36).substr(2);
+  const fn = "tmp-" + randomSuffix + "/test.txt";
+  writeFileSync(fn, data, 0o666);
```

Comment:
I don't want to add a test right now that writes into the current directory. I'd rather you comment this test out and add a TODO for later.

What about a failing test?  Writing to "/baddir/badfile.txt" should fail.

Reply:
How can you make sure that the test will fail? If the user has the right to create a file in this directory, the test will pass.

Reply:
`/baddir/` presumably doesn't exist... so it will fail